config/locales/de.yml

```diff
@@ -920,7 +920,7 @@ de:
     order_state:
       address: Adresse
       awaiting_return: erwartet Erstattung
-      canceled: Abgebrochen
+      canceled: Storniert
```

Comment:
I'm not sure this line should have changed google translate makes me _think_ it should remain the same at least the key should.